src/AngleSharp.Css/Converters/StandardValueConverter.cs

```diff
@@ -35,6 +35,11 @@ public ICssValue Convert(StringSource source)
                 source.SkipCurrentAndSpaces();
                 return source.ParseVar();
             }
+            else if (source.Content.IndexOf("/") != -1 && (ident.Isi(CssKeywords.Center) || ident.Isi(CssKeywords.Cover)))
```

Comment:
The standard converter will impact pretty much all rules - not just the `background` shorthand. The issue is for the `background` shorthand specifically, so this fix won't work - what we are looking for is the implementation of the rule, not just a bandaid that will actually introduce other issues.

Reply:
I went this way because fixing it in PointParser.cs would require having the CssBackgroundSizeValue() object having a public constructor so that internally its value can be "center / cover" or "cover / center". If you happy having the constructor changed for that object then I can put the code into the ParseSize() function of PointParser and modify that the CssBackgroundSizeValue() object as needed

Reply:
Yeah that sounds good - not sure though if the constructor needs to be public. I think what is missing right now is that the `ValueMode` can actually be independent for height and width - right now its just a single value.

Reply:
Sure, will look at that over the next couple of days see what I can do. If you want to keep that current constructor private will see what can be done to set the value to whats required.

Reply:
Changes pushed the move the code into PointParser() and CssBackgroundSizeValue() to handle this case